bindata/network-diagnostics/network-check-source.yaml

```diff
@@ -24,6 +24,7 @@ spec:
         app: network-check-source
         kubernetes.io/os: "linux"
     spec:
+      priorityClassName: system-cluster-critical
```

Comment:
I think we can use the `openshift-user-critical` priority for these daemonsets since these daemon sets are there only to check endpoints connectivity for debugging right ? They are not really more critical than the user-workloads. So we should be ok preempting them to accommodate the user load ?

Reply:
I believe that you are correct. Will make that change then open the pr